script/Arcueid/Day 4/04_00_ARC04_7_1.txt

```diff
@@ -121,7 +124,7 @@ Or are you telling me that you want me to go crazy?"
 [sha:127cf1413e050dc2f1295b5aab90e52fd3752832]{
 -- Page 648, Offset 7297.
 -- なんとなく、志貴が大切そうにしてるのがヤだったから」
-It's just that I didn't like how much you cherished them."
+Something about how much you cherished them rubbed me the wrong way is all."
```

Comment:
I think this is overwritten for how explicitly terse the Japanese is. I'd say revert